src/components/Select/Select.style.ts

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import { colors } from "@styles";
+
+const fonts = () => ({
```

Comment:
Change to object. it doesn't have any params and used to spread result into another objects